common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -1642,26 +1642,16 @@ def test_image_hint(self):
         assert not result['should_enable_next_hint']
 
     def test_demand_hint_logging(self):
-        def mock_location_text(self):  # lint-amnesty, pylint: disable=unused-argument
```

Comment:
@mraarif Are you referring to this? Yeah, that looks unrelated and due to mock versions. However, I do not see a reason for doing the mocking here (since we can test without it too) so I have removed it.

Reply:
this one was caused by the new version of `mock`, we upgraded a major version, I'd tag the author who added this mocking to get their point of view